### PR TITLE
Adjust query/event logging code in favour of more complex owncloud/di…

### PR DESCRIPTION
**Pull request by nickvergessen**
…agnostics (Downstream 27643)

* Adjust query/event logging code in favour of more complex owncloud/diagnostics
* Add descriptions to IQueryLogger and IEventLogger interfaces

### CODE REVIEW

File: lib/private/Diagnostics/QueryLogger.php [hunk 1]
```diff
@@ -3,7 +3,13 @@
  * @copyright Copyright (c) 2016, ownCloud, Inc.
  *
  * @author Morris Jobke <hey@morrisjobke.de>
+<<<<<<< HEAD
  * @author Robin Appelman <robin@icewind.nl>
+=======
+ * @author Robin Appelman <icewind@owncloud.com>
+ * @author Thomas Müller <thomas.mueller@tmit.eu>
+ * @author Piotr Mrowczynski <piotr@owncloud.com>
+>>>>>>> a5095447b7... Adjust query/event logging code in favour of more complex owncloud/diagnostics (#27643)
```
Comment (icewind1991) [inline lib/private/Diagnostics/QueryLogger.php]: conflict

File: lib/public/Diagnostics/IEventLogger.php [hunk 2]
```diff
@@ -3,7 +3,12 @@
  * @copyright Copyright (c) 2016, ownCloud, Inc.
  *
  * @author Morris Jobke <hey@morrisjobke.de>
+<<<<<<< HEAD
  * @author Robin Appelman <robin@icewind.nl>
+=======
+ * @author Robin Appelman <icewind@owncloud.com>
+ * @author Piotr Mrowczynski <piotr@owncloud.com>
+>>>>>>> a5095447b7... Adjust query/event logging code in favour of more complex owncloud/diagnostics (#27643)
```
Comment (icewind1991) [inline lib/public/Diagnostics/IEventLogger.php]: more conflict

File: lib/public/Diagnostics/IQueryLogger.php [hunk 3]
```diff
@@ -3,7 +3,12 @@
  * @copyright Copyright (c) 2016, ownCloud, Inc.
  *
  * @author Morris Jobke <hey@morrisjobke.de>
+<<<<<<< HEAD
  * @author Robin Appelman <robin@icewind.nl>
+=======
+ * @author Robin Appelman <icewind@owncloud.com>
+ * @author Piotr Mrowczynski <piotr@owncloud.com>
+>>>>>>> a5095447b7... Adjust query/event logging code in favour of more complex owncloud/diagnostics (#27643)
```
Comment (icewind1991) [inline lib/public/Diagnostics/IQueryLogger.php]: more

**Review by icewind1991 — CHANGES_REQUESTED** [review]
other than the unresolved conflicts 👍

**Comment by nickvergessen** [comment]
fixed

**Review by icewind1991 — APPROVED** [review]

**Comment by rullzer** [comment]
Updated the autoloader

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**